articles/native-platforms/ios-objc/index.yml

```diff
@@ -17,3 +17,12 @@ alias:
 seo_alias: ios-objc
 articles:
   - 01-login
+  - 02-custom-login
+  - 03-session-handling
+  - 04-user-profile
+  - 05-linking-accounts
+  - 06-rules
+  - 07-authorization 
+  - 08-calling-apis
+  - 09-mfa
+  
```

Comment:
Any `yml` file does need to have a final blank line, otherwise it won't work properly.